### PR TITLE
Added carousel example code

### PR DESCRIPTION
**Pull request by travisbmartin**

### CODE REVIEW

**Comment by jabdoa2** [comment]
Thanks for writing this down! Would it be possible to add this to the dev branch instead of latest? We can later merge it back from dev to latest if you want. The otherway around it might be missing in newer versions (like 0.51).

**Comment by travisbmartin** [comment]
jan,

Feel free to add it wherever you like! I'm still getting used to the process of contributing via github so bear with me.

File: game_logic/carousel.rst [hunk 1]
```diff
@@ -117,3 +117,134 @@ You can then use the carousel_<item>_selected event to start the mode that was s
...
         type: move_in
         direction: right
+        
+The following example is based around Bally's Doctor Who.  When the player starts a game, the player is shown via a carousel the option to pick eight modes, each representing a certain Doctor.  The flipper buttons control the carousel right and. left When the Launch Button is pressed, the game starts the mode selected by the player and launches the ball.
+
+.. code-block:: mpf-config
+
+    #config_version=5
+    
+    ##carousel.yaml
```
[the remark this answers is not in the document]
Comment (travisbmartin) [reply game_logic/carousel.rst]: Ok let's see if I can fix this properly

**Comment by jabdoa2** [comment]
rebased to dev in #183

**Pull request closed without merging**